### PR TITLE
Game revamp

### PR DESCRIPTION
**Pull request by Hugman76**
This PR contains the following:

- Update dependencies and their usages
- Make all text strings translatables instead of literals
- Lower the amount of max players on the 2 current exisiting map (31v1 is wayyy too many)
- Eliminating a player no longer sends a message (it would be too easy to spot who is the murderer then)

### CODE REVIEW

File: src/main/resources/data/murder_mystery/lang/en_us.json [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+    "text.murder_mystery.death_quote.17": "Hey! I saw this one!",
+    "text.murder_mystery.death_quote.18": "What are ya gonna do? stab me?",
+    "text.murder_mystery.death_quote.19": "Hey... you guys wanna see a dead body?",
+    "text.murder_mystery.death_quote.2": "Skibbity bop mm dada!",
+    "text.murder_mystery.death_quote.20": "10/10, would live again.",
+    "text.murder_mystery.death_quote.21": "Wow.",
+    "text.murder_mystery.death_quote.22": "It's beautiful, it's perfect, oh wow... just kidding.",
+    "text.murder_mystery.death_quote.3": "Oof",
+    "text.murder_mystery.death_quote.4": "Hey fellas!",
+    "text.murder_mystery.death_quote.5": "I did not get my Spaghetti-O's; I got spaghetti!",
```
Comment (Restioson) [inline src/main/resources/data/murder_mystery/lang/en_us.json:32]: Any reason this ordering is weird?

Comment (Hugman76) [reply src/main/resources/data/murder_mystery/lang/en_us.json:32]: Alphabetical JSON sortering did that of its own, did not notice it

**Review by SmellyModder — CHANGES_REQUESTED** [review]
These are some good changes 👍 
I'm unsure if the max players getting lowered is ideal, but it may be a good experiment.
I've also left some minor suggestions.

File: src/main/java/net/smelly/murdermystery/game/MMActive.java [hunk 2]
```diff
@@ -337,15 +316,14 @@ private void eliminatePlayer(ServerPlayerEntity attacker, ServerPlayerEntity pla
 		
 		this.scoreboard.updateRendering();
 		this.deadParticipants.add(player);
-		this.participants.sendMessage(player.getDisplayName().shallowCopy().append(" has been eliminated!").formatted(Formatting.RED));
-		this.participants.sendSound(SoundEvents.ENTITY_PLAYER_ATTACK_STRONG);
+		this.participants.sendSound(SoundEvents.ENTITY_PLAYER_ATTACK_CRIT);
```
[the remark this answers is not in the document]
Comment (Hugman76) [reply src/main/java/net/smelly/murdermystery/game/MMActive.java:319]: I thought that the previous sound was too low. I think that crit sounds better. Can revert it though

**Comment by Hugman76** [comment]
Thanks for the review! I made sure to correct almost everything you pointed out.
I left a comment on the sound change. I think modfifying it is better for the game's accessibility

File: src/main/java/net/smelly/murdermystery/game/MMConfig.java [hunk 3]
```diff
@@ -2,7 +2,6 @@
 
 import com.mojang.serialization.Codec;
 import com.mojang.serialization.codecs.RecordCodecBuilder;
-
```
Comment (SmellyModder) [inline src/main/java/net/smelly/murdermystery/game/MMConfig.java:5]: This small change should not be included, as it is irrelevant to the PR.

File: src/main/java/net/smelly/murdermystery/game/MMActive.java [hunk 4]
```diff
@@ -521,43 +499,53 @@ public String getInnocentsRemaining() {
...
+			return id;
+		}
+
+		public TranslatableText getName() {
+			return name;
+		}
+
+		public TranslatableText getWinMessage() {
+			return winMessage;
+		}
```
Comment (SmellyModder) [inline src/main/java/net/smelly/murdermystery/game/MMActive.java:547]: Style consistency.
```suggestion
		@Override
		public String toString() {
			return this.id;
		}

		public TranslatableText getName() {
			return this.name;
		}

		public TranslatableText getWinMessage() {
			return this.winMessage;
		}
```

**Review by SmellyModder — CHANGES_REQUESTED** [review]
Looking better now.
I've added some new minor suggestions after the last commit.
The suggested change in `MMActive` on line `171` was also not addressed in the last commit.

File: src/main/java/net/smelly/murdermystery/game/MMActive.java [hunk 5]
```diff
@@ -189,7 +168,7 @@ private void onOpen() {
 		
 		this.scoreboard.updateRendering();
 		this.spawnLogic.populateCoinGenerators();
-		this.participants.sendMessage(new LiteralText("Game will begin in 10 seconds!").formatted(Formatting.GREEN, Formatting.BOLD));
+		this.participants.sendMessage(new TranslatableText("text.murder_mystery.game_begin_in", 10).formatted(Formatting.GREEN, Formatting.BOLD));
```
[the remark this answers is not in the document]
Comment (haykam821) [reply src/main/java/net/smelly/murdermystery/game/MMActive.java:171]: It would probably be better to extract this into the configuration.

**Comment by Hugman76** [comment]
I've fixed the stuff you pointed out.
I followed haykam's suggested change in `MMActive` on line `171`, as it would prepare the game for multi-murderer game modes

**Review by SmellyModder — APPROVED** [review]

**Pull request merged**